ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-ref.yaml

```diff
@@ -62,6 +62,13 @@ ref:
       \[sig-operator\] an end user can use OLM can subscribe to the operator
       \[sig-storage\] In-tree Volumes \[Driver: nfs\] \[Testpattern: Dynamic PV (default fs)\] provisioning should provision storage with mount options
       \[sig-network\] Netpol \[LinuxOnly\] NetworkPolicy between server and client should stop enforcing policies after they are deleted
+      \[sig-cli] Kubectl client Simple pod should support exec through an HTTP proxy
+      \[sig-cli] Kubectl client Simple pod should support exec through kubectl proxy
+      \[sig-node] Pods should support retrieving logs from the container over websockets
+      \[sig-cli] Kubectl Port forwarding With a server listening on localhost should support forwarding over websockets
+      \[sig-cli] Kubectl Port forwarding With a server listening on 0.0.0.0 should support forwarding over websockets
+      \[sig-node] Pods should support remote command execution over websockets
```

Comment:
We should detect a proxy and skip these automatically: https://github.com/openshift/origin/pull/26398